examples/abitest/proto/abitest.proto

```diff
@@ -20,6 +20,7 @@ package oak.examples.abitest;
 
 message ABITestRequest {
   string filter = 1;
```

Comment:
Also rename here, if you follow the other suggestion.

Reply:
Done.